hooks/templatedataformatter.js

```diff
@@ -25,7 +26,8 @@ module.exports = function (pageMetadata, siteLevelAttributes, pageNameToConfig)
     params: currentLocaleConfig.params || {},
     relativePath,
     env: {
-      JAMBO_INJECTED_DATA: env.JAMBO_INJECTED_DATA
+      JAMBO_INJECTED_DATA: env.JAMBO_INJECTED_DATA,
+      packageJsonVersion: packageJson.version
     }
   };
   if (globalConfig.useJWT) {
```

Comment:
if useJWT is true, this would call getCleanedTemplateData() which would override env field. You may need to make some changes to ensure packageJsonVersion stay in env field

Reply:
updated! wish we had some automated test for jwt I don't really understand it too well